app/grandchallenge/challenges/templates/challenges/challenge_list.html

```diff
@@ -250,15 +250,15 @@ <h2>{{ year }}</h2>
                     {% for challenge in challenges %}
                         <div class="col-12 col-md-6 col-lg-4 mb-3 projectlink {{ challenge.filter_classes|join:" " }} {{ challenge.year }}">
                             <div class="card" style="height:400px">
-                                <a id="{{ challenge.short_name }}"
+                                <a id="{{ challenge.short_name }}" title="{{ challenge.title }}"
```

Comment:
```suggestion
                                <a id="{{ challenge.short_name }}" title="{% firstof challenge.title challenge.short_name %}"
```
Not all challenges will have titles, but they will have short names